node/config.go

```diff
@@ -199,6 +199,9 @@ type Config struct {
 
 	// JWTSecret is the path to the hex-encoded jwt secret.
 	JWTSecret string `toml:",omitempty"`
+
+	// EnablePersonal enables the deprecated personal namespace.
+	EnablePersonal bool `toml:",omitempty"`
```

Comment:
We should make sure this doens't end up in the config dump, otherwise removing it will invalidate all configs and Geth will fail to start.